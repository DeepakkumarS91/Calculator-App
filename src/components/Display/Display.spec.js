import React from 'react';
import Display from './Display';
import {shallow} from 'enzyme';

describe('Tests the Display',()=>
{let wrapper;
    
beforeEach(()=>wrapper=shallow(<Display displayValue= {'0'}/>));

it('Snapshot the Display',()=>{
    expect(wrapper).toMatchSnapshot()
})

it("Checks inside the Display that render <div/>",()=>{
    expect(wrapper.find('div').length).toEqual(1);
});

it("Checks the rendered value",()=>
{
    wrapper.setProps({displayValue:'2'});
    expect(wrapper.text()).toEqual('2')
})
 });