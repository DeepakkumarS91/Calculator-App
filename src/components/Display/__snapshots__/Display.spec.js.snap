// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tests the Display Snapshot the Display 1`] = `ShallowWrapper {}`;
